server/views/pages/rollout-status.njk

```diff
@@ -0,0 +1,44 @@
+{% extends "layout.njk" %}
+
+{% set pageTitle = applicationName + " - Rollout status" %}
+{% set pageId = 'rollout-status-page' %}
+
+{% block content %}
+
+    <div class="grid-row">
+        <div class="govuk-grid-column-three-quarters">
+            <h1 class="govuk-heading-l">Rollout status</h1>
+        </div>
+    </div>
+
+    <div class="grid-row govuk-body govuk-!-margin-bottom-4">
+        <div class="govuk-grid-column-three-quarters">
+            Sorry, your region is not one of the pilot areas for the service, as shown on the map below.
+            {% if authSource === 'delius' %}
+                We are currently running a pilot for probation officers in the <strong>NPS Wales</strong> region.
```

Comment:
Use `<span class='govuk-!-font-weight-bold'>` instead of `<strong>`

Reply:
Will do.